benchmarks/parse.sh

```diff
@@ -1,5 +1,6 @@
-#!/bin/sh
-
-cd benchmarks/output && go run ../parser/main.go -outdir ../csv && \
-cd ../csv && go run ../parser/main.go -distribution
+#!/usr/bin/env bash
 
```

Comment:
```suggestion
 set -e
```

Reply:
and so on for the other scripts that have sequences of actions that depend on each other.